central/cve/fetcher/manager_impl.go

```diff
@@ -16,66 +11,44 @@ import (
 	cveMatcher "github.com/stackrox/rox/central/cve/matcher"
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/sac"
 	pkgSearch "github.com/stackrox/rox/pkg/search"
 	"github.com/stackrox/rox/pkg/throttle"
 )
 
-var (
-	allAccessCtx = sac.WithAllAccess(context.Background())
-
-	connectionDropThrottle = throttle.NewDropThrottle(10 * time.Minute)
-)
-
 const (
 	minNewScannerReconcileInterval = 10 * time.Minute
 )
 
-type mode int
+var (
+	allAccessCtx = sac.WithAllAccess(context.Background())
+
+	connectionDropThrottle = throttle.NewDropThrottle(10 * time.Minute)
 
-const (
-	online = iota
-	offline
-	unknown
-	k8sIstioCveZipName = "k8s-istio.zip"
+	log = logging.LoggerForModule()
 )
 
-// Init copies build time CVEs to persistent volume
 func (m *orchestratorIstioCVEManagerImpl) initialize() {
-	if env.OfflineModeEnv.BooleanSetting() {
-		m.mgrMode = offline
-	} else {
-		m.mgrMode = online
-	}
-
-	if err := copyCVEsFromPreloadedToPersistentDirIfAbsent(); err != nil {
-		log.Errorf("could not copy preloaded Istio CVE files to persistent volume %q: %v", path.Join(persistentCVEsPath, commonCveDir, istioCVEsDir), err)
-		return
-	}
-	log.Infof("successfully copied preloaded CVE Istio files to persistent volume: %q", path.Join(persistentCVEsPath, commonCveDir, istioCVEsDir))
-
 	m.orchestratorCVEMgr.initialize()
 }
 
-// Fetch (works only in online mode) fetches new CVEs and reconciles them
+// Start begins the process to periodically scan orchestrator-level components.
 func (m *orchestratorIstioCVEManagerImpl) Start() {
-	if m.mgrMode != online {
-		log.Error("can't fetch in non-online mode")
-		return
-	}
-
-	ticker := time.NewTicker(fetchDelay)
+	ticker := time.NewTicker(env.OrchestratorVulnScanInterval.DurationSetting())
 	defer ticker.Stop()
```

Comment:
You can move this inside the anonymous func

Reply:
not "can". it's "must"! great catch. It is likely the ticket would have never gone off because it was stopped once `Start()` completed

Reply:
this proves I did not test this well enough to catch this. Will test this more to make sure it runs as expected